lib/util/identifier.js

```diff
@@ -305,15 +305,33 @@ exports.parseResource = (realFn => {
 
 /**
  * @param {string} filename the filename which should be undone
+ * @param {string} outputPath the output path that is restored (only relevant when filename contains "..")
  * @param {boolean} enforceRelative true returns ./ for empty paths
- * @returns {string} repeated ../ to leave the directory of the provided filename to be back on root dir
+ * @returns {string} repeated ../ to leave the directory of the provided filename to be back on output dir
  */
-exports.getUndoPath = (filename, enforceRelative) => {
+exports.getUndoPath = (filename, outputPath, enforceRelative) => {
```

Comment:
This signature change breaks this plugin
https://github.com/crimx/webpack-target-webextension/issues/13